smartfile/__init__.py

```diff
@@ -1,16 +1,17 @@
-from netrc import netrc
 import os
 import re
+import shutil
 import time
 import urllib
+
+from netrc import netrc
```

Comment:
netrc is very ancient and deprecated; also it has to do with FTP. What is it doing here?